frame/transaction-payment/src/lib.rs

```diff
@@ -260,10 +260,6 @@ pub mod pallet {
 		/// might be refunded. In the end the fees can be deposited.
 		type OnChargeTransaction: OnChargeTransaction<Self>;
 
-		/// The fee to be paid for making a transaction; the per-byte portion.
-		#[pallet::constant]
-		type TransactionByteFee: Get<BalanceOf<Self>>;
```

Comment:
This is a breaking change for all clients and UIs if they use this metadata constant. @ggwpez are any of our "metadata breaking checks" in place already? this PR is a good example. 

Although, I think most tools would use the `payment_info` RPC (cc @jacogr just in case)

Reply:
Indeed. Most usages should be via RPC - the logic should not be duplicated. (There will probably be breakages out in the wild where people do use this, but no reason to expose it if the runtime doesn’t use it)